cogs/tickets.py

```
@@ -75,46 +105,24 @@ async def on_raw_reaction_add(self, payload):
             icon_url=av,
             text="IsThicc Tickets"
         )
-        try:
-            dm_msg = await reaction_user.send(embed=priority_embed, timeout=10)
-            await dm_msg.add_reaction("🟢")  # Green
-            await dm_msg.add_reaction("🟡")  # Yellow
-            await dm_msg.add_reaction("🔴")   # Red
-
-            def priority_check(reaction, user):
-                emoji = str(reaction.emoji)
-                return user.id == reaction_user.id and reaction.channel == reaction_user.dm_channel \
-                       and emoji == "🟢" \
-                       or emoji == "🟡" \
-                       or emoji == "🔴"
-
-            reaction, user = await self.bot.wait_for("reaction_add", check=priority_check)
-
-            if str(reaction.emoji) == "🟢":
-                priority = ["Low", "🟢"]
-            elif str(reaction.emoji) == "🟡":
-                priority = ["Medium", "🟡"]
-            else:
-                priority = ["High", "🔴"]
-
-            topic_embed = em(
-                title="What do you want your ticket to be about?",
-                colour=discord.Colour.green(),
-                timestamp=datetime.utcnow()
-            )
-            topic_embed.set_footer(
-                icon_url=av,
-                text="IsThicc Tickets"
-            )
-            dm_msg = await reaction_user.send(embed=topic_embed, timeout=60)
-            await dm_msg.add_reaction("🟢")  # Green
-            await dm_msg.add_reaction("🟡")  # Yellow
-            await dm_msg.add_reaction("🔴")   # Red
 
-            def topic_check(m):
-                return user.id == reaction_user.id and user.id == reaction_user.dm_channel
+        priority_message = await reaction_user.send(embed=priority_embed)
```

Add a catch for this, so it doesn't silently fail to send the message if the user has DMs off.